recipe/bld.bat

```diff
@@ -0,0 +1,36 @@
+set CMAKE_CONFIG=Release
+
+mkdir build_shared_%CMAKE_CONFIG%
+pushd build_shared_%CMAKE_CONFIG%
+
+cmake -G "NMake Makefiles" ^
+      -DCMAKE_BUILD_TYPE:STRING=%CMAKE_CONFIG% ^
+      -DBUILD_SHARED_LIBS:BOOL=ON ^
+      "%SRC_DIR%"
+nmake
+ctest -V
+
+move bro.exe "%LIBRARY_BIN%\"
+move *.dll "%LIBRARY_BIN%\"
```

Comment:
Yes, usually .exe and .dll files are placed in bin directory; All .lib files are placed in lib. That's common for Windows.
Others conda-forge's feedstocks are doing similarly (for instance, curl's exe is copied to bin; boost's dlls are copied to bin too ).